assets/css/style.css

```diff
@@ -0,0 +1,136 @@
+/* font styling */
+h1,
+h2,
+h3,
+h4,
+p,
+ul,
+li {
+font-family: 'Open Sans', sans-serif;
+}
+p {
+  font-size: 1.25em;
+  color: gray;
+  line-height: 32px;
+}
+
+h1 {
+  font-size: 40px !important;
+  color: black;
+}
+
+.white-text p{
```

Comment:
Let's rethink `.white-text p` and `.white-text h1` for a bit:

Both of those are basically the exact same code. Any time you see duplicate code, it's probably worth thinking of a generic way to apply it. If you also want some `<li>` and `<a>` tags to have white text, will you add 2 more `white-text` classes?

Try and think of a way to have 1 generic class called `white-text` that you can add to any tag. Thoughts?